Resources/Prototypes/Species/slime.yml

```
@@ -36,19 +36,19 @@
       points: 1
       required: false
     Chest:
-      points: 1
+      points: 4 # imp
       required: false
     Legs:
-      points: 4
+      points: 8 # imp
       required: false
     Arms:
-      points: 4
+      points: 48 # imp
```

```suggestion
      points: 8 # imp 4<8
```
I'm assuming this was an error.
Note original value for future maintenance work.

---

whoops